src/a-frame/component/tour.js

```diff
@@ -80,10 +80,28 @@ export default {
     this.animate(target)
   },
 
+  setCameraMode: function (mode) {
+    if (['person', 'bird'].indexOf(mode) < -1) {
+      console.error('not supported camera mode: ' + mode)
+      return
+    }
+    this._isPlaying = false
+    var pos = clone(this.el.getAttribute('position'))
+    var rot = clone(this.el.getAttribute('rotation'))
+    pos.y = mode === 'person' ? 1.4 : 7
```

Comment:
I noticed I never mentioned my principle of avoiding magic numbers (which I break myself, sometimes, because I'm making mistakes as well).

General rule of thumb: If a number has a meaning, express that as a constant:

```javascript
  pos.y = mode === 'person' ? HEIGHT_PERSON : HEIGHT_BIRDS_EYE
```

reduces the mental overhead to figure out what the numbers mean